web-components/src/components/tabs/IconTab.tsx

```diff
@@ -7,17 +7,20 @@ import type { Theme } from 'src/theme/muiTheme';
 import { TextSize } from '../typography/sizing';
 import { Subtitle } from '../typography/Subtitle';
 import { RegenTab } from './';
+import { LinkComponentProp } from './IconTabs';
 
 export interface IconTabProps extends RegenTab {
   icon?: JSX.Element;
   hidden?: boolean;
   size?: TextSize;
+  href?: string;
+  linkComponent?: LinkComponentProp;
   sxInner?: SxProps<Theme>;
 }
 
 const StyledTab = styled(Tab, {
   shouldForwardProp: prop => prop !== 'size',
-})<TabProps & { size: TextSize }>(({ theme, size }) => ({
+})<TabProps & { href?: string; size: TextSize }>(({ theme, size }) => ({
```

Comment:
`href` should already be typed inside `TabProps` but for some reason, TS does not find it so I had to explicitly add it.